exercises/hello-world/hello_world.cpp

```diff
@@ -0,0 +1,16 @@
+#include "hello_world.h"
+
+// Use everything from the 'std' namespace.
+// This lets us write 'string' instead of 'std::string'.
+using namespace std;
+
+namespace hello_world {
+
+// Define the function itself. This could have also been written as 'std::string
+// hello_world::hello()'.
```

Comment:
It might help to put
```c++
std::string hello_world::hello()
```
all on a single line.

Reply:
Done